src/openrct2-ui/windows/Ride.cpp

```diff
@@ -4032,7 +4032,7 @@ namespace OpenRCT2::Ui::Windows
                 return;
 
             uint16_t reliability = ride->reliabilityPercentage;
-            widgetProgressBarSetNewPercentage(widgets[WIDX_RELIABILITY_BAR], std::max<uint8_t>(10, reliability));
```

Comment:
You're misinterpreting this. It means the value will be 10 or the actual reliability, whichever is _higher_. It makes 10 the minimum, rather than the maximum.